src/wp-includes/class-wp-oembed.php

```diff
@@ -212,6 +210,7 @@ public function __construct() {
 		 * | Photobucket  | photobucket.com      | 2.9.0     | 5.1.0     |
 		 * | Funny or Die | funnyordie.com       | 3.0.0     | 5.1.0     |
 		 * | CollegeHumor | collegehumor.com     | 4.0.0     | 5.3.1     |
+ 		 * | Hulu         | hulu.com		 | 2.9.0     | 5.4.2	 |
```

Comment:
Hey @tacitonic Thanks again for working on this. 

It looks like the PHP linter that's run on Travis picked up a couple of minor issues related to the white space here:
https://travis-ci.com/github/WordPress/wordpress-develop/jobs/362005117

Looks like the comment should be indented using tabs, but all other spacing within the table should be spaces?

I would maybe try to replicate what's on the other lines 😄

Reply:
I did use both tab as well as space indentation, as only tabs didn't align with the existing pipes. I'll see if I can use tabs exclusively. :)

Reply:
It seems that spaces were used rather than tabs.